addons/mail/models/mail_mail.py

```diff
@@ -231,6 +231,15 @@ def send(self, auto_commit=False, raise_exception=False):
             smtp_session = None
             try:
                 smtp_session = self.env['ir.mail_server'].connect(mail_server_id=server_id)
+            except Exception:
+                exc_class, exc, tb = sys.exc_info()
```

Comment:
Yep, probably. I was testing various things. Not sure it is the correct way to use odoo exceptions.